services.yaml

```diff
@@ -3078,6 +3105,42 @@ definitions:
         items:
           type: string
           example: MyApp
+  JVM:
+    description: JVM properties
+    type: object
+    properties:
```

Comment:
i think we should create a unique id attribute to an agent, so that we can one day make some POST on it with /agents/{agent-id} such as DORMANT.